Updates to the Tokelau and NZ empire updates

stepps00 opened the pull request:
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/348.

This PR includes:

- A new Tokelau Islands `dependency` record, with verified concordances and concordance properties. This supersedes the Tokelau Islands `locality` record. The hierarchy/parent_id/belongsto values match the close-by NZ `dependency`, [Niue](https://whosonfirst.mapzen.com/spelunker/id/85632493/).
- Added a `edtf:deprecated` date, `mz:is_current:0` property, and `wof:superseded_by`value to the Tokelau Islands record
- Updated the New Zealand `empire` record to include the new Tokelau Islands geometry.
- Exportified all records

New geometry:

<img width="828" alt="tokelau" src="https://cloud.githubusercontent.com/assets/18567700/19169737/b9c2efb4-8be3-11e6-92d6-570e808bf695.png">

nvkelso commented on data/115/882/147/3/1158821473.geojson:
```diff
@@ -0,0 +1,75 @@
+    "qs:photos_9r":"0",
+    "qs:photos_sr":"0",
+    "qs:pop_sr":"3",
+    "src:geom":"unknown",
```
Is this from `mz`?

nvkelso commented on data/136/253/053/136253053.geojson:
```diff
@@ -4,11 +4,11 @@
-    "geom:bbox":"-177.9579972,-52.600313271,178.843923373,-8.543226821",
-    "geom:latitude":-41.797885,
-    "geom:longitude":171.060632,
+    "geom:area":0.003433,
```
This is extremely suspicious – too small, coordinates look wrapped to maybe only include the new geom instead of also the old geom?

stepps00 commented:
All comments were addressed in 2eb7b9f.

- Corrected new `empire` geometry and exportified, which fixed the `geom:` values
- Added `mz` as the new geom source for 1158821473.geojson

nvkelso commented:
Generally this looks correct now... but the empire geom is using the lower resolution Natural Earth shape instead of the GIT LFS managed high rez shape + other dependencies. 

Please ensure each child (country, dependency) of NZ empire has a rev-geo point inside this new empire geom (that's also inside whatever the child geom is per child).

stepps00 commented:
https://github.com/whosonfirst-data/whosonfirst-data/pull/946#issuecomment-337998384 addressed in 785ca57.

`reversegeo:*` lat/long properties were added to the three dependency records under the NZ empire - Cook Islands, Tokelau, and Niue. The lat/long points fall within each dependency shape and the NZ empire shape.

The country record of New Zealand is OK as-is and did not need new lat/long props.

stepps00 commented:
This was merged to master through the commits above, closing issue. 

Tokelau: https://github.com/whosonfirst-data/whosonfirst-data/blob/master/data/115/882/147/3/1158821473.geojson
